code/modules/clothing/head/misc.dm

```diff
@@ -278,6 +278,22 @@
 		rad = 0
 	)
 
+/obj/item/clothing/head/onestar
+	name = "One Star officer cap"
+	desc = "A fancy red and blue cap sporting the One Star insignia. its made out of a strange material that feels like it could stop a bullet."
```

Comment:
```suggestion
	desc = "A fancy red and blue cap sporting the One Star insignia. It's made out of a strange material that feels like it could stop a bullet."
```